lib/stream.js

```diff
@@ -72,14 +67,36 @@ module.exports.encrypt = async ctx => {
   } else {
     logger.debug('The stream is encrypted.');
   }
-};
+}
+
+async function getShards(props) {
+  const { logger, client, streamName } = props;
+  logger.debug(`Retrieving shards for the "${streamName}" stream…`);
 
-module.exports.tag = async ctx => {
-  const { client, logger, streamName: StreamName, tags } = ctx;
+  const params = { StreamName: streamName, MaxResults: 1000 };
```

Comment:
Is MaxResults a required field? What about pagination if it goes over?

Reply:
I just removed it. The default for the list shards call is still 1,000. I guess supporting more than 1,000 shards will be a future feature.